### ISSUE TITLE
ColorSwitcherActivity crash upon layer switch

### ISSUE THREAD

**Issue by langsmith** [mapbox/mapbox-android-demo#1090]
Was doing `8.1.0-alpha.1` QA and saw that `ColorSwitcherActivity` crashes when the spinner menu is switched from `Water` to `Building`

```
Process: com.mapbox.mapboxandroiddemo.debug, PID: 26753
    com.mapbox.mapboxsdk.exceptions.ConversionException: Not a valid rgb/rgba value
        at com.mapbox.mapboxsdk.utils.ColorUtils.rgbaToColor(ColorUtils.java:136)
        at com.mapbox.mapboxsdk.style.layers.FillLayer.getFillColorAsInt(FillLayer.java:214)
        at com.mapbox.mapboxandroiddemo.examples.styles.ColorSwitcherActivity$1.onItemSelected(ColorSwitcherActivity.java:54)
        at android.widget.AdapterView.fireOnSelected(AdapterView.java:944)
```
![Screen Shot 2019-06-07 at 10 25 29 AM](https://user-images.githubusercontent.com/4394910/59122107-bcb47000-890e-11e9-8c12-032a13e25e89.png)

**Comment by langsmith** [comment]
No crash for me on `master`. Looking into what happens on `8.1.0-alpha.1` and/or `8.1.0-alpha.2`.

**Comment by langsmith** [comment]
Crashing on `8.1.0-alpha.2`

Seems that `fill-color: rgba(233.00001525878907,233.00001525878907,231.00001525878907,1)` can't be converted to an int 🤔 . `building.getFillColorAsInt()` leads to the 💥 

![Screen Shot 2019-06-07 at 1 34 23 PM](https://user-images.githubusercontent.com/4394910/59132062-fd20e780-8928-11e9-9cc6-688bd6b593ff.png)
![Screen Shot 2019-06-07 at 1 34 16 PM](https://user-images.githubusercontent.com/4394910/59132063-fd20e780-8928-11e9-89be-dbe9b9902f96.png)

**Comment by langsmith** [comment]
Same crash on `8.1.0-alpha.1`

**Comment by langsmith** [comment]
cc @mapbox/maps-android as fyi that this could be something to look into before the stable `8.1.0` release.

**Comment by zugaldia** [comment]
@langsmith good find - I added it to the board, I agree to prioritize.

**Comment by langsmith** [comment]
`ls-maps-sdk-8.1.0`  is the QA branch where this crash is happening

**Comment by LukasPaczos** [comment]
Nice catch @langsmith!

https://github.com/mapbox/mapbox-gl-native/pull/14954 is on its way, and I'm going to verify with the snapshot when it lands.

**Comment by LukasPaczos** [comment]
Fixed with https://github.com/mapbox/mapbox-gl-native/pull/14954 that's going to be included in the `v8.1.0`.
